src/Razor/Microsoft.CodeAnalysis.Razor/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,26 @@
+#nullable enable
```

Comment:
Does this mean this project has nullable enabled (because I don't think it does): https://github.com/dotnet/aspnetcore/blob/master/src/Razor/Microsoft.CodeAnalysis.Razor/src/Microsoft.CodeAnalysis.Razor.csproj#L3-L9

Reply:
No, means the analyzer will handle nullable annotations if they exist in the assembly